MetaMask/metamask-extension
# chore: Update lavamoat to a version with more diff-friendly policy ordering

## **Description**

This lavamoat update brings a different sorting comparator for policy.json files that will produce more readable diffs.
This PR has 2 commits - one that reorders the policy without making any changes and another that updates lavamoat packages on top of that.

For better clarity on the policy.json files, inspect each commit separately.

This is going to be hard to review and merge because conflict resolution requires a redo. Gonna have to schedule it carefully.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/29311?quickstart=1)

## **Related issues**


## **Manual testing steps**

1. run a local policy update and verify the diff is not ridiculously large (that would mean the update failed to apply properly after policy got reordered here)

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
